entgql/extension.go

```diff
@@ -188,6 +193,14 @@ func WithWhereFilters(b bool) ExtensionOption {
 	}
 }
 
+// WithSchemaGenerator add a hook for generate GQL schema
+func WithSchemaGenerator() ExtensionOption {
```

Comment:
💯

That means users can use it even if they don't use gqlgen.

Reply:
Yes, but it require some refactor for WhereInput feature to working as the same way